notebooks/HASP/WavelengthAdjustment/WavelengthAdjustment.ipynb

```diff
@@ -0,0 +1,2632 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #25.    with fits.open(f"{cos_unshifted_coadd}/{coadd_cos}") as hdul:
</pre><p>No need for an f string here:</p>

<pre class="ql-syntax" spellcheck="false">	with fits.open(cos_unshifted_coadd /coadd_cos) as hdul:
			wl_coadd..
			....
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>